react-app/src/components/SupportPage/contactHelpDesk.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const ContactHelpDesk: React.FC = () => {
```

Comment:
```suggestion
const ContactHelpDesk = () => {
```

`React.FC` is [deprecated](https://github.com/typescript-cheatsheets/react#function-components)